Plugin.php

```diff
@@ -27,7 +27,9 @@ public function registerComponents()
             'RainLab\Blog\Components\Post'       => 'blogPost',
             'RainLab\Blog\Components\Posts'      => 'blogPosts',
             'RainLab\Blog\Components\Categories' => 'blogCategories',
-            'RainLab\Blog\Components\RssFeed'    => 'blogRssFeed'
+            'RainLab\Blog\Components\RssFeed'    => 'blogRssFeed',
+	    'RainLab\Blog\Components\ArchiveList'  => 'blogArchiveList',
```

Comment:
Missing space here

Reply:
and there's an extra space in between `'RainLab\Blog\Components\ArchiveList'` and the `=>`